packages/@aws-cdk/aws-servicecatalogappregistry/test/application.test.ts

```diff
@@ -256,7 +255,7 @@ describe('Application', () => {
 
       Template.fromStack(stack).hasResourceProperties('AWS::RAM::ResourceShare', {
         AllowExternalPrincipals: false,
-        Name: 'RAMShare5bb637032063',
+        Name: 'RAMSharee6e0e560e6f8',
```

Comment:
Why has this been changed?

Reply:
I don't know why the unique ID is changed in this unit test. And it seems all the unique IDs are changed.

Reply:
This is because the hash is calculated from the [application object](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-servicecatalogappregistry/lib/application.ts#L162). If the application object structure changes, the hash will change.